speculative_decoding/model/model.py

```diff
@@ -47,40 +48,42 @@ def __init__(self, **kwargs):
             raise NotImplementedError("openai-compatible")
 
     def load(self):
-        tensor_parallel_count = self._config["model_metadata"].get(
-            "tensor_parallelism", 1
-        )
-        pipeline_parallel_count = self._config["model_metadata"].get(
-            "pipeline_parallelism", 1
-        )
+        if not helpers.is_triton_server_alive():
+            tensor_parallel_count = self._config["model_metadata"].get(
+                "tensor_parallelism", 1
```

Comment:
nit: Let's be consistent re: what we call these variables and the structure for the `config.yaml` so that they're all the same across Truss'. I started on this with the template work, we can tackle this in a follow-up.